### DISCUSSION TITLE
Is Copilot still using Codex as it's underlying model

### DISCUSSION THREAD

**Discussion by Formgren** [community/community#119915] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Forgive me if this is an ignorant question but I am having a hard time understanding the current stage of the underlying model that Github Copilot uses.
When i check the logs of the copilot chat it calls both GPT-3.5 turbo and GPT-4 api. My understanding is that Codex have been updated with GPT-4, does this mean that Copilot is still using Codex but further enhanced with GPT-4 or is it simply an adaption over GPT-4 now?

**Comment by davecheney** [comment, marked as answer]
Sadly the `copilot-codex` name is a legacy of a poor decision (on my part) to encode the brand of the model in the url. We do not use the codex model any more.

**Reply by Formgren** [reply]
Thank you for the answer! @davecheney
